### PR TITLE
[14.0][MIG] fieldservice_sale_recurring

### PR DESCRIPTION
**Pull request by Freni-OSI**

### CODE REVIEW

File: fieldservice_sale_recurring/models/sale_order_line.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    )
+
+    def _field_create_fsm_recurring_prepare_values(self):
+        self.ensure_one()
+        template = self.product_id.fsm_recurring_template_id
+        note = self.name
+        if template.description:
+            note += "\n " + template.description
+        return {
+            "customer_id": self.order_id.partner_id.id,
```
Comment (brian10048) [inline fieldservice_sale_recurring/models/sale_order_line.py:25]:
```suggestion
```
there is no customer_id on fsm.recurring

File: fieldservice_sale_recurring/models/sale_order_line.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+            "location_id": self.order_id.fsm_location_id.id,
+            "start_date": self.order_id.expected_date,
+            "fsm_recurring_template_id": template.id,
+            "description": note,
+            "max_orders": template.max_orders,
+            "fsm_frequency_set_id": template.fsm_frequency_set_id.id,
+            "fsm_order_template_id": template.fsm_order_template_id.id,
+            "sale_id": self.order_id.id,
```
Comment (brian10048) [inline fieldservice_sale_recurring/models/sale_order_line.py:33]:
```suggestion
```
There is no sale_id on fsm.recurring

**Review by brian10048 — CHANGES_REQUESTED** [review]

**Comment by max3903** [comment]
@Freni-OSI Please fix or close.

**Comment by Freni-OSI** [comment]
Closing as merged another PR in v14.

**Pull request closed without merging**